tests/unittests/Foundation/NSJSONSerializationTests.m

```diff
@@ -124,13 +124,13 @@ void VerifyDataWithJSONObjectThrows(id testObject) {
     id testObject6 = @{ @"foo" : @"1", @"bar" : @"2" };
     id testObject7 = @{ @"foo" : @{ @"bar" : @"1" }, @"foo2" : @{ @"bar2" : @"2" } };
     id testObject8 = [NSUUID UUID];
-    id testObject9 = @{ @"key" : @"value", @"nullkey" : [NSNull null], @"array" : @[ @{ @"subnullkey" : [NSNull null] } ] };
```

Comment:
We are already using a fully tested third party json parser - the windows json serializer, so I think you are both right - we could live with the simplification of the test.

---
In reply to: [80983725](https://github.com/Microsoft/WinObjC/pull/1043#discussion_r80983725) [](ancestors = 80983725,80978644,80960041,80824620)